content/articles/statistical-hypothesis-testing-python-implementation/index.md

```diff
@@ -0,0 +1,506 @@
+In this case, encountering type 1 error is assuming she does not have fibroid when she actually is, i.e. the null hypothesis is true but we reject it.
+If a type two error is encountered, which is assuming she's pregnant when she is truly not, the null hypothesis is wrong but we insist on not rejecting it. The consequence might probably be leaving the bump-like pregnancy to grow, and it turns out to be an acute fibroid.
+
+Considering these two situations, it would be safe to conclude that the consequences of making a type-1-error is much more grave than making a type-2-error. While trying to solidify the assumption, it is mandatory to employ the service of a p-value, which is basically used to determine whether to accept or reject our null hypothesis.
+
+### 4. [Confidence of Null Hypothesis (Probability Value)](#4-confidence-of-null-hypothesis)
+
+[P-value](https://www.investopedia.com/terms/p/p-value.asp) explains the likelihood of the gathered fact or data to occur under the condition of the null hypothesis.
+Technically, the only way we can accept or reject our ground truth (null hypothesis) is after determining our [*P-value*](https://www.scribbr.com/statistics/p-value/) which is an abbreviation for probability value. It is the only thing that has the decisive power to accept or reject H0. 
+
+***The smaller our P-values is the more delicate it is to accept our null hypothesis.***
+
+To calculate our P-value, there should be a certain level of significance to our ground truth or initial assumption, so a smaller P-value is telling us there are more reasons why we should accept our alternative hypothesis.
+
+P-value is within the range of 0 and 1, but the threshold value is always set at 0.05, the threshold value can also be called *alpha*, alpha is always set before the experiments in order to avoid bias. 
+Alpha is the level we set our p-value at inorder to ascertain whether to accept our null hypothesis or not.
+Our alpha value is the probability of making a wrong decision when the ground truth(H0) is true. It is always calculated as 
+```
+alpha = 1 - confidence level
+```
+if the confidence level from what we've seen in our data is 95%, then 
+```
+alpha = 1 - 0.95
+```
+which is 0.05.
+
+Why is 95% confidence level the most accustomed?
+
+Since we know that alpha level is the probability of encountering a Type I error(which is rejecting H0 despite being true), would it not make sense to make it as bitsy as possible. Imagine our alpha value being set to 10%, obviously, there is an increasing chance that we will make the mistake of rejecting the null hypothesis, but it must not be too small as well, if we set the alpha value as 1%, it will be too small that our decision will be reeking of bias. Hence, the standard has been 5%.
+
+While inplementing statistical hypothesis testing, there are some methodologies that can be employed, in this tutorial, I will be touching on three methods of carrying out statistical hypothesis testing.
```

Comment:
It is very evident from this sentence that you haven't used Grammarly.
The sentence can be edited as follows:
```
While implementing statistical hypothesis testing, there are some methodologies that can be employed. In this tutorial, I will be touching on three methods of carrying out statistical hypothesis testing.
```